source/by-role/developer/common/page-and-component-controllers.rst

```diff
@@ -1,17 +1,17 @@
 :is-up-to-date: True
-:last-update: 4.1.2
+:last-updated: 4.1.6
 
-.. index:: Content Controllers, Content Type Controllers, Page Interceptors
+.. index:: Page Controllers, Component Controllers, Page Interceptors
 
-.. _content-type-controllers:
+.. _page-and-component-controllers:
 
----------------------------------------------------------
-Content Type Controller (Page and Component Interceptors)
----------------------------------------------------------
+----------------------------------------------------------------
+Page and Component Controllers (Page and Component Interceptors)
```

Comment:
Are interceptors different than controllers? If not, let's stick to calling them one thing, in this case, controllers.